.gitignore

```diff
@@ -121,5 +121,6 @@ packages/**/babel.config.json
 .jest-with-debug/
 #intelij management files
 .idea
+.scannerwork
```

Comment:
I'd be interested to know how your local code quality checks have been set up. Things like this should benefit the whole team. Maybe you could do a show and tell at one of the tech team calls or direct others in the team to resources you used?

Reply:
.idea is my IDE. I normally remove it from commit, but this time I failed to notice it. I have removed it in the subsequent commit by adding it to .gitignore.
.scannerwork helps me run sonar scanner before I commit. This way, I remove sonar noise from commit. It does not work 100%